Source/WebCore/html/HTMLSelectElement.cpp

```diff
@@ -456,8 +456,10 @@ ExceptionOr<void> HTMLSelectElement::setItem(unsigned index, HTMLOptionElement*
         return { };
     }
 
-    if (index > maxSelectItems - 1)
-        index = maxSelectItems - 1;
+    if (index >= length() && index >= maxSelectItems) {
+        document().addConsoleMessage(MessageSource::Other, MessageLevel::Warning, makeString("Blocked attempt to expand the option list and set an option at index. The maximum list length is ", index, maxSelectItems, '.'));
```

Comment:
Why are we outputting index and maxSelectItems here.
We should probably just output maxSelectItems since we're saying "the maximum length is" prior to those two numbers.

Reply:
Sure, I will remove "index". Also any changes in Console Message?

Reply:
Updated.